src/main/proto/wfa/panelmatch/protocol/crypto/oblivious_query.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.panelmatch.protocol;
+
+option java_package = "org.wfanet.panelmatch.protocol";
+option java_multiple_files = true;
+
+// An argument in every Request proto. It contains various knobs for tuning the system.
+message Parameters {
+}
+
+// A request to get a public/private key pair
+message GenerateKeysRequest {
+    Parameters parameters = 1;
+}
+
+message GenerateKeysResponse {
+    bytes public_key = 1;
+    bytes private_key = 2;
+}
+
+// Used in the request to encrypt a set of queries
+message ShardQueryBucketIdTriplet {
```

Comment:
What about just naming this `UnencryptedQuery`?

Reply:
sure